### ISSUE TITLE
Still highlights when emmet-insert-flash-time is negative

### ISSUE THREAD

**Issue by angrybacon** [smihica/emmet-mode#98]
`emacs --version`

> GNU Emacs 25.3.1
> Copyright (C) 2017 Free Software Foundation, Inc.
> GNU Emacs comes with ABSOLUTELY NO WARRANTY.
> You may redistribute copies of GNU Emacs
> under the terms of the GNU General Public License.
> For more information about these matters, see the file named COPYING.

`emacs -q`

```elisp
(setq package-load-list '((emmet-mode t) (epl t) (pkg-info t)))
(package-initialize)
(setq emmet-insert-flash-time -1)
```

`(pkg-info-package-version 'emmet-mode)`

```elisp
(20161113 2158)
```

**Comment by ianpan870102** [comment]
Yes, this happens to me too. Currently I'm just setting it to 0.01, so it blinks only a tiny bit.
